code/hello-world-tutorial/hello.tape

```diff
@@ -0,0 +1,11 @@
+Output hello.gif
+Set Theme { "name": "Aardvark Blue", "black": "#191919", "red": "#aa342e", "green": "#4b8c0f", "yellow": "#dbba00", "blue": "#1370d3", "purple": "#c43ac3", "cyan": "#008eb0", "white": "#bebebe", "brightBlack": "#454545", "brightRed": "#f05b50", "brightGreen": "#95dc55", "brightYellow": "#ffe763", "brightBlue": "#60a4ec", "brightPurple": "#e26be2", "brightCyan": "#60b6cb", "brightWhite": "#f7f7f7", "background": "#102040", "foreground": "#dddddd", "cursor": "#007acc", "selection": "#bfdbfe" }
```

Comment:
I'd like to make all the colors the same for every gif in the book. This is great to start! Should we pick these colors for the other gifs as well?

Reply:
Yeah. I really like this theme (somewhat biased though).

I have an open PR on VHS to import all the themes from their upstream source (iterm2colorschemes->windowsterminalthemes->vhs) which makes this much simpler: `Set Theme "Aaardvark Blue"`. Let's wait a bit for that to happen.

Reply:
https://github.com/charmbracelet/vhs/pull/377 was just merged so it is actually possible now.
```shell
go install github.com/charmbracelet/vhs@main
~/go/bin/vhs hello.tape
```
```plain
File: hello.tape
Host your GIF on vhs.charm.sh: vhs publish <file>.gif
Output .gif hello.gif
Set Theme Aardvark Blue
Set Width 800
Set Height 450
Type cargo run
Enter 1
Sleep 7s
Screenshot hello.png
Sleep 1s
Type q
Sleep 2s
Creating hello.gif...
```